superset-frontend/jest.config.js

```diff
@@ -26,7 +26,10 @@ module.exports = {
     '^spec/(.*)$': '<rootDir>/spec/$1',
   },
   testEnvironment: 'enzyme',
-  setupFilesAfterEnv: ['jest-enzyme', '<rootDir>/spec/helpers/shim.ts'],
+  setupFilesAfterEnv: [
+    '<rootDir>/node_modules/jest-enzyme/lib/index.js',
+    '<rootDir>/spec/helpers/shim.ts',
+  ],
```

Comment:
@nytai any reason for this change?

Reply:
I was having issues with `shim.ts` not loading when I was running tests in vscode, I was digging through some docs and most of them were using absolute paths. Is it causing issues?